README.md

```diff
@@ -143,6 +143,7 @@ Apps people use when taking transit.
 - [OpenTripPlanner Android](https://github.com/CUTR-at-USF/OpenTripPlanner-for-Android/wiki) - An Android app for [OpenTripPlanner](http://www.opentripplanner.org/)
 - [OpenTripPlanner iOS](https://github.com/opentripplanner/OpenTripPlanner-iOS) - An iOS app for [OpenTripPlanner](http://www.opentripplanner.org/)
 - [Transportr](https://github.com/grote/Transportr) An Android app that uses [public-transport-enabler](https://github.com/schildbach/public-transport-enabler) in order to connect to many different transport networks worldwide.
+- [Öffi](https://gitlab.com/oeffi/oeffi) (Android) - The King of Public Transit!
```

Comment:
Could you please change this to a more descriptive and objective summary of the app?  Where it's deployed, what it does, the language it's written in, etc. would be useful.

Reply:
What about this?

"Öffi tells you where and when trains and busses go, including delays and replacement bus service for more and more transport authorities in Europe and beyond. It's written in Java and installed on 10M+ devices."

I won't sign CLAs though, so probably someone else needs to PR this.